includes/Media.php

```diff
@@ -258,10 +259,8 @@ public function get_callback_media_source( $prepared ) {
 		$id = $prepared['id'];
 
 		$terms = wp_get_object_terms( $id, self::STORY_MEDIA_TAXONOMY );
-		if ( is_array( $terms ) && $terms ) {
-			$term = array_shift( $terms );
-
-			return $term->slug;
+		if ( is_array( $terms ) && ! empty( $terms ) ) {
+			return array_shift( $terms )->slug;
```

Comment:
Why was this line changed? This is unrelated change. Please do this in another PR.